autoscaler/controllers/gateway/config/jaeger.go

```diff
@@ -25,23 +25,23 @@ func (j *Jaeger) DestType() common.DestinationType {
 	return common.JaegerDestinationType
 }
 
-func (j *Jaeger) ModifyConfig(dest *odigosv1.Destination, currentConfig *commonconf.Config) {
+func (j *Jaeger) ModifyConfig(dest *odigosv1.Destination, currentConfig *commonconf.Config) error {
 
 	if !isTracingEnabled(dest) {
 		log.Log.V(0).Error(ErrorJaegerTracingDisabled, "skipping Jaeger destination config")
```

Comment:
Since we now return the error to the caller, we can choose to have the log printing only once in `root.go` and remove a lot of unnecessary code from specific destinations